### PR TITLE
plan section content with js

### PR DESCRIPTION
**Pull request by Dovile-dote**

### CODE REVIEW

File: js/pages/home.js [hunk 1]
```diff
@@ -1,5 +1,10 @@
 // component imports
 
+import { renderPlans } from '../components/plansList/plansList.js';
+console.log(renderPlans);
```
Comment (Paskanas) [inline js/pages/home.js]: reikia nepalikti.

**Review by Paskanas — CHANGES_REQUESTED** [review]

File: js/pages/home.js (hunk 1 above)
Comment (Dovile-dote) [reply js/pages/home.js]: ops

Comment (Dovile-dote) [reply js/pages/home.js]: pataisiau

**Review by Paskanas — APPROVED** [review]
All good

**Review by Kristle22 — APPROVED** [review]

**Pull request merged**